webapp/src/Controller/Jury/ContestController.php

```diff
@@ -95,6 +96,10 @@ public function indexAction(Request $request): Response
             if (!$contest) {
                 throw new NotFoundHttpException('Contest not found');
             }
+            if ($contest->isLocked()) {
```

Comment:
I think unfreeze, finalize and delay_start should all be possible when its locked? Or we should get a popup 'do you want to unlock and do $action` as the user is already in the UI where the risk is much lower.

Reply:
Yep, you are right. Locking a contest is mostly meant to prevent accidental edits to problem data and such. We *need* to allow the start time interface to allow pausing/resuming contest countdown via the CDS anyway. So I relaxed things a little.